docs/releases/1.10.txt

```diff
@@ -24,6 +24,21 @@ recommend** and only officially support the latest release of each series.
 What's new in Django 1.10
 =========================
 
+Maximum size of a request body is now limited
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
These headings need to use dash (-------) instead of ~~~~~~~~ to fix the docs build.